File: Programming Reference/API Study/Projects/Unit_2-Video_6/explore.py
import os
from twilio.rest import Client

account_sid = os.environ['TWILIO_ACCOUNT_SID']
auth_token = os.environ['TWILIO_AUTH_TOKEN']

client = Client(
    account_sid,
    auth_token
)

#for msg in client.messages.list():
#    print(f"Deleting {msg.body}")
#    msg.delete()

msg = client.messages.create(
    to="+16618002221",
    from_="+12695254066",
    body="Hello from Python",
)

print(f"Created a new message: {msg.sid}")

File: Programming Reference/API Study/Projects/Unit_2-Video_5/explore.js
require("dotenv").config(); // Loads Environment Variables

// Unit 2: Video 5: Using Helper Libraries (JavaScript)
// This is a Comment
/* This is a C-Type Comment*/

const Twilio = require("twilio");   // Access Client

const accountSid = process.env.TWILIO_ACCOUNT_SID;
const authToken = process.env.TWILIO_AUTH_TOKEN;

//const client = require('twilio')(accountSid, authToken);

const client = new Twilio(  // Setup a New Client (Account SID, Auth Token)
    accountSid,
    authToken 
);


client.messages
    .list() //  Makes a get request
    .then(messages =>
        console.log(`The most recent message is ${messages[0].body}`)    // messages is an array
    ).catch(err => console.error(err));

console.log('Gathering your message log');
